lib/msal-browser/docs/v2-migration.md

```diff
@@ -0,0 +1,60 @@
+# Migrating from MSAL 2.x to MSAL 3.x
+
+If you are new to MSAL, you should start [here](initialization.md).
+
+If you are coming from [MSAL v1.x](../../msal-core/), you should check [this guide](v1-migration.md) first to migrate to [MSAL v2.x](../../msal-browser/).
```

Comment:
Can we have someone hop from v1 to v3?

Reply:
I think it is totally possible, isn't it?

Reply:
That's what the v2 guide suggests now. First check the v1 guide and then follow next steps.